docs/src/components/ComponentStatus/index.tsx

```diff
@@ -63,11 +63,7 @@ export const ComponentStatusList = () => (
   <>
     {Object.values(Groups).map(group => (
       <React.Fragment key={group}>
-        <HeadingWithLink>
-          <Heading as="h2" type="title2">
-            {group}
-          </Heading>
-        </HeadingWithLink>
+        <H2>{group}</H2>
```

Comment:
There are a bunch of these changes because these headings need to end up in the table of contents.